include/deal.II/grid/grid_tools.h

```diff
@@ -454,8 +454,9 @@ namespace GridTools
    * the transform() function above, so the requirements on the triangulation
    * stated there hold for this function as well.
    */
+  template <int dim>
```

Comment:
can you please leave a ``@note`` here that this function is only supported for dim==2?

Reply:
I changed this. I have written only one ``=`` for consistency with the function below where it is written similarly.